server/api/cart.js

```diff
@@ -0,0 +1,49 @@
+const router = require("express").Router();
+const { models } = require("../db/index.js");
+const { Cart, Product } = models;
+//temporary get route
+router.get("/", (req, res, next) => {
```

Comment:
move these into the user routes. so we can get a cart based on user on /api/user/:userId/cart